build/media-suite_compile.sh

```diff
@@ -1223,13 +1223,14 @@ if [[ $mp4box = "y" ]] && do_vcs "https://github.com/gpac/gpac.git"; then
 fi
 
 _check=(SvtHevcEnc.pc libSvtHevcEnc.dll.a svt-hevc/EbApi.h
-    bin-video/Svt-Hevc/{libSvtHevcEnc.dll,SvtHevcEncApp.exe})
+    bin-video/{libSvtHevcEnc.dll,SvtHevcEncApp.exe})
 if [[ $svthevc = "y" ]] && do_vcs "https://github.com/OpenVisualCloud/SVT-HEVC.git"; then
-    do_uninstall bin-video/SVT-Hevc/libSvtHevcEnc.dll.a "${_check[@]}"
+    do_uninstall bin-video/libSvtHevcEnc.dll.a "${_check[@]}"
```

Comment:
```suggestion
    do_uninstall "${_check[@]}" include/svt-hevc
```
Only leave legacy installation directories/files to uninstall if they were at some point used in the suite. If `.dll.a` was never installed there other than on @1480c1's tests, there's no point leaving it. The include dir, on the other hand, is not being completely cleaned.